### PR TITLE
Update messages_es_ES.properties

### PR DESCRIPTION
**Pull request by NeilJared**
Updated Spanish translation

# License Agreement for Contributions
By submitting this pull request, I acknowledge and agree that my contributions will be included in Stirling-PDF and that they can be relicensed in the future under MPL 2.0 (Mozilla Public License Version 2.0) license. 

(This does not change the general open-source nature of Stirling-PDF, simply moving from one license to another license)

### CODE REVIEW

File: src/main/resources/messages_es_ES.properties [hunk 1]
```diff
@@ -434,15 +434,15 @@ addPageNumbers.selectText.4=Número de inicio
 addPageNumbers.selectText.5=Páginas a numerar
 addPageNumbers.selectText.6=Texto personalizado
 addPageNumbers.customTextDesc=Texto personalizado
-addPageNumbers.numberPagesDesc=Qué páginas numerar, por defecto 'all', también acepta 1-5 o 2,5,9 etc
```
Comment (Frooodle) [inline src/main/resources/messages_es_ES.properties]: These are key words that the code looks for, you can't change the all tag or the filename tag etc

File: src/main/resources/messages_es_ES.properties [hunk 2]
```diff
@@ -434,15 +434,15 @@ addPageNumbers.selectText.4=Número de inicio
 addPageNumbers.selectText.5=Páginas a numerar
 addPageNumbers.selectText.6=Texto personalizado
 addPageNumbers.customTextDesc=Texto personalizado
-addPageNumbers.numberPagesDesc=Qué páginas numerar, por defecto 'all', también acepta 1-5 o 2,5,9 etc
-addPageNumbers.customNumberDesc=Por defecto a {n}, también acepta 'Página {n} de {total}', 'Texto-{n}', '{filename}-{n}
```
Comment (Frooodle) [inline src/main/resources/messages_es_ES.properties]: Same for filename

**Review by Frooodle — CHANGES_REQUESTED** [review]

**Review by NeilJared — COMMENTED** [review]
Ok, change them all so, or tell me if I must do that

**Review by NeilJared — COMMENTED** [review]
Done

**Comment by NeilJared** [comment]
I think it's done now
cheers!

**Review by Frooodle — APPROVED** [review]

**Pull request merged**